elastalert/ruletypes.py

```diff
@@ -535,17 +535,18 @@ def get_all_terms(self, args):
         else:
             end = ts_now()
         start = end - window_size
-        if self.rules.get('use_strftime_index'):
-            index = format_index(self.rules['index'], start, end)
-        else:
-            index = self.rules['index']
-        time_filter = {self.rules['timestamp_field']: {'lte': dt_to_ts(end), 'gte': dt_to_ts(start)}}
-        query_template['filter'] = {'bool': {'must': [{'range': time_filter}]}}
-        query = {'aggs': {'filtered': query_template}}
+        step = datetime.timedelta(**self.rules.get('window_step_size', {'days': 1}))
```

Comment:
Since you're adding this new rule property, you should update the documentation too!